rootfs/api/models/app.py

```diff
@@ -43,7 +44,7 @@ def validate_app_structure(value):
         if any(int(v) < 0 for v in value.values()):
             raise ValueError("Must be greater than or equal to zero")
     except ValueError as err:
-        raise ValidationError(err)
+        raise ValidationError(str(err))
```

Comment:
Yeah, ValidationError is both Django and DRF, the latter augments it a bit to better play with its View logic